src/components/PackagesList/PackagesList.tsx

```diff
@@ -0,0 +1,86 @@
+import Box from "@mui/system/Box"
+import * as React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+
+interface PackagesListProps {
+    children: React.ReactNode;
+}
+
+interface PackageDistribution {
+    id: number;
+    name: string;
+    address: string;
+    postcode: string;
+    city: string;
+    country: string;
+    lat: string;
+    lng: string;
+}
+
+const fetchData = () => {
+    const resp = fetch("https://santa.deployed.space/api/distribution");
+    return resp;
+}
+
+
+
+const PackagesList = ({ children }: PackagesListProps) => {
```

Comment:
It seems that this component does not use `children`, please remove it if it is unnecessary